src/natcap/invest/coastal_blue_carbon/coastal_blue_carbon.py

```diff
@@ -662,7 +662,8 @@ def execute(args):
     total_accumulation_rasters = {}  # Across all 3 pools
     total_accumulation_tasks = {}
     prior_transition_year = baseline_lulc_year
-    for current_transition_year in sorted(transition_years) + [analysis_year]:
+    for current_transition_year in sorted(
+            transition_years.union(set([analysis_year]))):
```

Comment:
`transition_years.add(analysis_year)` would be slightly more succinct, but this is also just fine!

Reply:
In terms of making it more succinct, I hadn't realized that the extra `set()` isn't needed in order to take the union of two sets, which I've patched in 3be2bb4c.  It should at least be a bit more readable for the creation of a one-off set for use just in this one case.

I wish I could just use `add` here!  That would be lovely ... but as it stands, `set.add` modifies the set but doesn't return the new set.

Let me know what you think @emlys !

Reply:
Oh, you're right about `set.add`! This looks great.